### DISCUSSION TITLE
Upload proof of Student Developer Pack Application

### DISCUSSION THREAD

**Discussion by savio-mb** [community/community#145917] [GitHub Education]
### Hi everyone,

Hi everyone, is there any other option to take a picture in my webcam or phone? The camera of my phone and webcam is poor. I have the proof in pdf but upload doesnt work

**Comment by GabJuarez** [comment]
Upload it as an image. I had a similar problem when submitting my college ID photo. What I did was take a screenshot of my personal information from the college website, and GitHub Education accepted it for the application. :)

**Reply by savio-mb** [reply]
Hello Gabriel, How upload the image? There is no option to upload in the screen.

**Reply by ArielFSantos** [reply]
me the same, no had option to upload, only take a picture. :/

**Reply by GabJuarez** [reply]
> Hello Gabriel, How upload the image? There is no option to upload in the screen.

Could you share a screenshot here to see what can we do?
Btw if you can mark my reply as an answer it would help me a lot 😸

**Comment by BhathiyaPrasad** [comment]
No Brother uploading it via pdf is having less chance to approve find a mobile and upload the proofs
